cmasher/tests/test_utils.py

```diff
@@ -65,6 +66,88 @@ def _MPL38_colormap_eq(cmap, other) -> bool:
 
 
 # %% PYTEST CLASSES AND FUNCTIONS
+# Pytest class for combine_cmaps
+class Test_combine_cmaps:
+    # Test if multiple colormaps can be combined with names
+    def test_combine_cmaps_with_names(self):
+        cmaps = ["Blues", "Oranges", "Greens"]
+        nodes = [0.25, 0.75]
+        combined_cmap = combine_cmaps(cmaps, nodes, n_rgb_levels=256)
+
+        blues_cmap = mpl.colormaps["Blues"]
+        oranges_cmap = mpl.colormaps["Oranges"]
+        greens_cmap = mpl.colormaps["Greens"]
+        assert np.allclose(combined_cmap(0.0), blues_cmap(0))
+        assert np.allclose(combined_cmap(0.25), oranges_cmap(0))
+        assert np.allclose(combined_cmap(0.75), greens_cmap(0))
+        assert np.allclose(combined_cmap(1.0), greens_cmap(255))
+        assert combined_cmap.N == 256
+
+    # Test if multiple matplotlib colormaps can be combined
+    def test_combine_mpl_cmaps(self):
+        blues_cmap = mpl.colormaps["Blues"]
+        oranges_cmap = mpl.colormaps["Oranges"]
+        greens_cmap = mpl.colormaps["Greens"]
+
+        cmaps = [blues_cmap, oranges_cmap, greens_cmap]
+        nodes = [0.25, 0.75]
+        combined_cmap = combine_cmaps(cmaps, nodes, n_rgb_levels=256)
```

Comment:
Let's avoid repetition by combining both tests (I did the formatting by hand, linters might still complain)
```suggestion
    # Test if multiple colormaps can be combined
    @pytest.mark.parametrize(
        "cmaps", [
        	["Blues", "Oranges", "Greens"],
        	[mpl.colormaps["Blues"], mpl.colormaps["Oranges"], mpl.colormaps["Greens"]],
        ]
    )
    def test_combine_cmaps_with_names(self, cmaps):
        nodes = [0.25, 0.75]
        combined_cmap = combine_cmaps(cmaps, nodes, n_rgb_levels=256)

        blues_cmap = mpl.colormaps["Blues"]
        oranges_cmap = mpl.colormaps["Oranges"]
        greens_cmap = mpl.colormaps["Greens"]
        assert np.allclose(combined_cmap(0.0), blues_cmap(0))
        assert np.allclose(combined_cmap(0.25), oranges_cmap(0))
        assert np.allclose(combined_cmap(0.75), greens_cmap(0))
        assert np.allclose(combined_cmap(1.0), greens_cmap(255))
        assert combined_cmap.N == 256

    # Test if multiple matplotlib colormaps can be combined
    def test_combine_mpl_cmaps(self):
        blues_cmap = mpl.colormaps["Blues"]
        oranges_cmap = mpl.colormaps["Oranges"]
        greens_cmap = mpl.colormaps["Greens"]

        cmaps = [blues_cmap, oranges_cmap, greens_cmap]
        nodes = [0.25, 0.75]
        combined_cmap = combine_cmaps(cmaps, nodes, n_rgb_levels=256)
```

Reply:
Thanks for the detailed reviewing. And sorry I should have used `parametrize`.

Reply:
no need to apologise ! and feel free to take your time, I'm not in a rush :-)